src/main/java/wooteco/subway/controller/PathController.java

```diff
@@ -1,33 +1,28 @@
 package wooteco.subway.controller;
 
+import javax.validation.Valid;
+
 import org.springframework.http.ResponseEntity;
-import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RestController;
 
 import wooteco.subway.controller.dto.PathRequest;
 import wooteco.subway.controller.dto.PathResponse;
-import wooteco.subway.domain.Path;
-import wooteco.subway.domain.Station;
 import wooteco.subway.service.PathService;
-import wooteco.subway.service.StationService;
 
-@Controller
+@RestController
 public class PathController {
 
-    private final StationService stationService;
     private final PathService pathService;
 
-    public PathController(StationService stationService, PathService pathService) {
-        this.stationService = stationService;
+    public PathController(PathService pathService) {
         this.pathService = pathService;
     }
 
     @GetMapping("/paths")
-    public ResponseEntity<PathResponse> findPath(@ModelAttribute PathRequest pathRequest) {
-        Station source = stationService.findOne(pathRequest.getSource());
-        Station target = stationService.findOne(pathRequest.getTarget());
-        Path path = pathService.findPath(source, target);
-        return ResponseEntity.ok().body(PathResponse.from(path));
+    public ResponseEntity<PathResponse> findPath(@ModelAttribute @Valid PathRequest pathRequest) {
```

Comment:
```suggestion
    public ResponseEntity<PathResponse> findPath(@Valid PathRequest pathRequest) {
```

1단계에서 생략 피드백 드렸었어요 ㅎㅎ

Reply:
이 부분은 수정했다가 @Valid 를 추가하면서 어떻게 익셉션을 잡을까 이것 저것 시도하다가 다시 붙여 놓고 제거를 안했군요 😭 꼼꼼하게 확인하겠습니다.